fastcov.py

```diff
@@ -460,12 +460,12 @@ def getSourceLines(source, fallback_encodings=[]):
         return f.readlines()
 
 # Returns whether source coverage changed or not
-def exclProcessSource(fastcov_sources, source, exclude_branches_sw, include_branches_sw, fallback_encodings):
+def exclProcessSource(fastcov_sources, source, exclude_branches_sw, include_branches_sw, exclude_line_marker, fallback_encodings):
     # Before doing any work, check if this file even needs to be processed
     if not exclude_branches_sw and not include_branches_sw:
         # Ignore unencodable characters
         with open(source, errors="ignore") as f:
-            if "LCOV_EXCL" not in f.read():
+            if not exclude_line_marker and "LCOV_EXCL" not in f.read():
```

Comment:
```
file_str = f.read()
if not any(marker in file_str for marker in exclude_line_marker):
    return False
````

Reply:
We would ignore other LCOV exclusion options (LCOV_EXCL_START/STOP/BR_LIN) then.

Reply:
That's true. The idea of this conditional is to exit without processing file line by line if the file does not contain any exclusion markers. What do you think about:

```
    # Before doing any work, check if this file even needs to be processed
    if not exclude_branches_sw and not include_branches_sw:
        # Ignore unencodable characters
        with open(source, errors="ignore") as f:
            fileStr = f.read()
            markerFound = False
            for marker in exclude_line_marker + ["LCOV_EXCL"]:
                if marker in fileStr:
                    markerFound = True
                    break

            if not markerFound:
                return False
```